### DISCUSSION TITLE
Bearer Token Auth type support

### DISCUSSION THREAD

**Discussion by wingyplus** [darrenburns/posting#137] [Ideas]
I want to propose a Bearer Token auth type to `Auth` menu. Like Postman does: 

<img width="1082" alt="Screenshot 2567-11-13 at 10 54 05" src="https://github.com/user-attachments/assets/3bf21341-8420-4c2c-9a27-1c7059484c54">

Currently, I work around this by manually setting the `Authorization` header in the' Headers' menu. Having this support would shorten the typing flow a little.

**Comment by edgarrmondragon** [comment]
I've started a PR: https://github.com/darrenburns/posting/pull/152

**Comment by darrenburns** [comment]
Added in 2.5.0, out now!

Thanks @edgarrmondragon for implementing this :)
